routes/auth.js

```diff
@@ -0,0 +1,97 @@
+const router = require('express').Router();
+const bcrypt = require('bcryptjs');
+const User = require('../models/User.model');
+const mongoose = require('mongoose');
+
```

Comment:
On the lab instruction they tell you to create a route that you can only access if you're logged in. For that you need to require the middleware:
`const { isLoggedIn, isLoggedOut } = require("../middleware/route-guard");`